Jabkit preferences does not work.

### JabRef version

Latest development branch build (please note build date below)

### Operating system

GNU / Linux

### Details on version and operating system

Endeavour OS; kernel:  6.16.7-arch1-1 

### Checked with the latest development build (copy version output from About dialog)

- [x] I made a backup of my libraries before testing the latest development version.
- [x] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

`jabkit preferences [export | import | reset]` 
Then you get an exception.

```
picocli.CommandLine$InitializationException: Cannot instantiate org.jabref.cli.Preferences$PreferencesExport: the class has no constructor
	at picocli.CommandLine$DefaultFactory.create(CommandLine.java:5693)
	at picocli.CommandLine$Model$CommandUserObject.getInstance(CommandLine.java:12296)
	at picocli.CommandLine$Model$CommandSpec.userObject(CommandLine.java:6445)
	at picocli.CommandLine$Interpreter.clear(CommandLine.java:13553)
	at picocli.CommandLine$Interpreter.parse(CommandLine.java:13599)
	at picocli.CommandLine$Interpreter.processSubcommand(CommandLine.java:13938)
	at picocli.CommandLine$Interpreter.processArguments(CommandLine.java:13834)
	at picocli.CommandLine$Interpreter.parse(CommandLine.java:13620)
	at picocli.CommandLine$Interpreter.processSubcommand(CommandLine.java:13938)
	at picocli.CommandLine$Interpreter.processArguments(CommandLine.java:13834)
	at picocli.CommandLine$Interpreter.parse(CommandLine.java:13620)
	at picocli.CommandLine$Interpreter.parse(CommandLine.java:13588)
	at picocli.CommandLine$Interpreter.parse(CommandLine.java:13483)
	at picocli.CommandLine.parseArgs(CommandLine.java:1552)
	at picocli.CommandLine.execute(CommandLine.java:2173)
	at org.jabref.JabKit.main(JabKit.java:99)
	at JabKitLauncher.main(JabKitLauncher.java:45)
Caused by: java.lang.NoSuchMethodException: org.jabref.cli.Preferences$PreferencesExport.<init>()
	at java.base/java.lang.Class.getConstructor0(Class.java:3218)
	at java.base/java.lang.Class.getDeclaredConstructor(Class.java:2493)
	at picocli.CommandLine$DefaultFactory.create(CommandLine.java:5664)
	at picocli.CommandLine$DefaultFactory.create(CommandLine.java:5691)
        ... 16 more
```

it appears that the Preferences class defines non static inner classes, and thus no constructor can be called for them from the static Preferences.

Should be an easy fix. 

### Appendix

...
<details>

<summary>Log File</summary>

```
Paste an excerpt of your log file here
```

</details>

--- a/jabkit/src/main/java/org/jabref/cli/Preferences.java
+++ b/jabkit/src/main/java/org/jabref/cli/Preferences.java
@@ -58,11 +58,11 @@ class Preferences implements Runnable {
     @Command(name = "import", description = "Import preferences from a file.")
     static class PreferencesImport implements Callable<Integer> {
 
-        @ParentCommand
-        private Preferences parent;
-
         @Parameters(index = "0", arity = "1", description = "The file to import preferences from.")
         Path inputFile;
+
+        @ParentCommand
+        private Preferences parent;
 
         @Override
         public Integer call() {
@@ -79,11 +79,11 @@ class Preferences implements Runnable {
     @Command(name = "export", description = "Export preferences to a file.")
     static class PreferencesExport implements Callable<Integer> {
 
-        @ParentCommand
-        private Preferences parent;
-
         @Parameters(index = "0", arity = "1", description = "The file to export preferences to.")
         Path outputFile;
+
+        @ParentCommand
+        private Preferences parent;
 
         @Override
         public Integer call() {

